package.json

```diff
@@ -21,50 +21,51 @@
   },
   "dependencies": {
     "chalk": "^5.2.0",
-    "lit": "^2.7.2"
+    "lit": "^3.0.0"
   },
   "peerDependencies": {
-    "@alaskaairux/design-tokens": "^3.13.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.3.0"
+    "@alaskaairux/design-tokens": "^3.15.5",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.3"
   },
   "devDependencies": {
-    "@alaskaairux/design-tokens": "^3.13.0",
-    "@alaskaairux/icons": "^4.25.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.3.0",
-    "@commitlint/cli": "^17.6.1",
-    "@commitlint/config-conventional": "^17.6.1",
-    "@open-wc/testing": "^3.1.8",
-    "@rollup/plugin-node-resolve": "^15.0.2",
+    "@alaskaairux/design-tokens": "^3.15.5",
```

Comment:
Do we need to keep the old tokens around?

Reply:
See lines https://github.com/AlaskaAirlines/auro-icon/pull/95/files#diff-c0a5ddd5b365322730802e8cb15c95c3b6e965cbd3e569c69a2f089753844cdbR13-R18

Once we release the updated auro icons and update this repo with the new version, then we can delete.